rayon-core/src/sleep/README.md

```diff
@@ -216,4 +216,4 @@ Meanwhile, the sleepy thread does the following:
 Either PushFence or SleepFence must come first:
 
 * If PushFence comes first, then PushJob must be visible to ReadJob.
-* If SleepFence comes first, then IncSleepers is visible to ReadSleepers.
\ No newline at end of file
+* If SleepFence comes first, then IncSleepers is visible to ReadSleepers.
```

Comment:
What is the change here?

Reply:
Hi, I checked this out for you. The change here is the addition of a final newline - or in other words, a `\n` is now the last byte in the file.

I'm not exactly sure why it showed up like this - maybe it's just some github jank?

In any case, it's possible this change was performed automatically when @Bilogweb3 saved the file - at the very least, I remember VSCode doing this for me when making markdown files.